grai-server/app/init.py

```diff
@@ -11,10 +12,21 @@
 username = config("DJANGO_SUPERUSER_USERNAME", None)
 password = config("DJANGO_SUPERUSER_PASSWORD", None)
 email = config("DJANGO_SUPERUSER_EMAIL", None)
+workspace_name = config("DJANGO_SUPERUSER_WORKSPACE", None)
 
 if not username and email:
     username = email
 
+user = None
+
 if username and password and not User.objects.filter(is_superuser=True).exists():
     print(f"Creating superuser {username}")
-    User.objects.create_superuser(username, password)
+    user = User.objects.create_superuser(username, password)
+
+if workspace_name and not Workspace.objects.filter(name=workspace_name).exists():
```

Comment:
This comment isn't specific to this line of code but the current logic permits users to be created orphaned from a workspace and I'm not sure if that was intended. I think we want every user to belong to a workspace even if we ultimately delete the workspace, change an affiliation, or rename it.

Reply:
It isn't that easy to ensure that each user is in at least one workspace, we would need to ensure the presence of a `Membership`.  I think we should handle the case that a user is not a member of any Workspace, as this is likely to happen at some point, unless we are going to prevent users being removed from their final workspace, which doesn't make a whole lot of sense.

How we handle creating the first superuser, through this script, is however a slightly different question.  I could change the logic to only create a superuser if `DJANGO_SUPERUSER_WORKSPACE` is also present, to require all three environment variables.

Reply:
> unless we are going to prevent users being removed from their final workspace, which doesn't make a whole lot of sense.

Good point.

> How we handle creating the first superuser, through this script, is however a slightly different question. I could change the logic to only create a superuser if DJANGO_SUPERUSER_WORKSPACE is also present, to require all three environment variables.

The current implementation makes perfect sense if users can be intentionally orphaned from workspaces but in that scenario I would actually prefer to create a default workspace (`default` or w/e) for the initial superuser rather than requiring an additional parameter. The previous logic was to create superusers only when the system didn't yet have a superuser though so we might want to move the workspace creation logic under the username/password conditional in that case as well. As an aside there's a nifty `objects.get_or_create` method we can use to avoid the workspace filtered `.exists` as well.

Reply:
Done, you can override the default workspace name with a parameter if required.